### ISSUE TITLE
Add YAML support

### ISSUE THREAD

**Issue by minecrafter** [minecrafter/voxelwind#22]
Per https://forums.voxelwind.com/t/what-about-snakeyaml-library/37, `jackson-dataformat-yaml` will be added to the Voxelwind API. `voxelwind-api` already includes Jackson.

~~**This will be added once the 0.16 update branch is merged into master.**~~

**Comment by minecrafter** [comment]
I have decided to resume work on master now that I have some more details on 0.16's release.
